tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
+    "target": "es6",
+    "module": "esnext",
```

Comment:
is "esnext" stable, or will it change underneath our feet from build to build?

Reply:
Understanding what the tsconfig module option does is non-trivial, but it appears have the module option as esnext should not change the output (while setting the target as esnext would).